examples/relationships/README.md

```diff
@@ -0,0 +1,209 @@
+# Gatsby Relationships Example
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/27d162a0-612b-4d8f-8382-40eee4f6b5d9/deploy-status)](https://app.netlify.com/sites/kontent-gatsby-example-relationships/deploys)
+
+Following example sites is showcasing the possibilities of [Gatsby Foreign keys Fields](https://www.gatsbyjs.com/docs/schema-customization/#foreign-key-fields) on some simple scenarios.
+
+## Get started
+
+⚠ Before running any of the following scripts locally ensure the site is using the latest dependencies
+
+```sh
+# In the root folder of this repo
+yarn # install all required packages
+yarn build # build the latest version of the local packages
+```
+
+### Develop site
+
+```sh
+# open the /example/relationships folder
+cd examples/relationships
+yarn develop # runs `gatsby develop` command
+```
+
+Now you could browse the site on <http://localhost:8000> and see GraphiQL explorer on <http://localhost:8000/___graphql>.
+
+## Content modeling
+
+This section explains how the content is modeled. You could follow to next section ["Import the content to your on Kontent project"](#Import-site-content-to-your-Kontent-project) and explore the models by your own as well as the sample data based on it. Or you could create models manually to familiarize yourself with the Kontent user interface.
+
+Once you create the content models, you could create content items based on these and the site would be capable to handle the content and render it.
+
+### Kontent content models
+
+Kontent project contains a simple article content base, that could be tagged. There are two content types `Article` and `Tag`. Kontent project is using two languages `en-US` and `cs-CZ` to showcase [Language variant relationships](#Language-variant-relationships);
+
+#### Article content type
+
+This content type - `Article item` - has following structure:
+
+- Title - **Text** element
+- Content - **Text** element
+- Date - **Date & Title** element
+- Slug - **URL slug** element - auto-generated from title
+- Tags - **Linked items** element - content type limited to `Tag`
+
+This content type represents an article, first four elements (`Title`, `Content`, and `Date`) are meant to be used for some sample content that should be displayed on site. The `Content` element is set to simple text because of simplicity. Rich text element type possibilities are described in [Resolution example](../resolution#readme). The `Slug` element is used to route registration. `Tags` element is modeled as Linked items element type because of [Linked from relationship](#Linked-from-relationship) example.
```

Comment:
```suggestion
This content type represents an article, the first four elements (`Title`, `Content`, and `Date`) are meant to be used for some sample content that should be displayed on site. The `Content` element is set to simple text because of simplicity. Rich text element type possibilities are described in the [Resolution example](../resolution#readme). The `Slug` element is used to route registration. The `Tags` element is modeled as Linked items element type because of [Linked from relationship](#Linked-from-relationship) example.
```